src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.WindowsAzure.MobileServices;
+using WebJobs.Extensions.EasyTables;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
```

Comment:
Add some descriptive comments on each of these samples, since we'll link to these in the wiki. See TimerSamples for an example.